loop_frequency: 100
cycle_time_error_threshold: 0.002
thread_priority: 95

joint_names: ["yaw_joint", "pitch_joint", "roll_joint"]
init_pose: [1.31, 0.085, 2.0]
init_vel: [1.0, 1.0, 1.0]
init_kp: [0.05, 0.05, 0.05]
init_kd: [0.025, 0.025, 0.025]